Firestore/core/src/firebase/firestore/api/query_core.h

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
Question: just curious, why is this file called `query_core` and not just `query`?